modules/installation-generate-aws-user-infra-ignition.adoc

```diff
@@ -61,6 +62,49 @@ Because you create and manage the worker machines yourself, you do not need
 to initialize these machines.
 endif::[]
 
+ifeval::["{context}" == "installing-gcp-user-infra"]
+. Remove the Kubernetes manifest files that define the worker machines:
++
+----
+rm -f openshift/99_openshift-cluster-api_worker-machineset-*.yaml
+----
++
+Because you create and manage the worker machines yourself, you do not need
+to initialize these machines.
+
+. Currently, emptying the compute pools makes control plane machines schedulable. However, due to a link:https://github.com/kubernetes/kubernetes/issues/65618[Kubernetes limitation], router Pods running on control plane machines will not be reachable by the ingress load balancer.
++
+Ensure the `mastersSchedulable` parameter is set to `false` in the
+`manifests/cluster-scheduler-02-config.yml` scheduler configuration file to keep
+router Pods and other workloads off the control plane machines.
+
+. Optional: If you do not want
+link:https://github.com/openshift/cluster-ingress-operator[the Ingress Operator]
+to create DNS records on your behalf, remove the `privateZone` and `publicZone`
+sections from the `manifests/cluster-dns-02-config.yml` DNS configuration file:
++
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: DNS
+metadata:
+  creationTimestamp: null
+  name: cluster
+spec:
+  baseDomain: example.openshift.com
+  privateZone: <1>
+    tags:
+      Name: mycluster-100419-mmhhx-int
+      kubernetes.io/cluster/mycluster-100419-mmhhx: owned
+  publicZone: <1>
+    id: F3UR364WQ91KFV
+status: {}
+----
+<1> Remove these sections completely.
++
+If you do so, you must add ingress DNS records manually in a later step.
+endif::[]
```

Comment:
Since we can't `xref` from within modules, I've had to add this as an "Additional resource" at the end of the section a few lines before this context.

Reply:
lgtm